src/App/Fossa/Telemetry/Types.hs

```diff
@@ -0,0 +1,122 @@
+module App.Fossa.Telemetry.Types (
```

Comment:
`App.Fossa.Telemetry.Types` and `App.Fossa.Telemetry.Utils` are modules split horizontally.  We should probably try to split them vertically instead.

Reference post: https://www.haskellforall.com/2021/05/module-organization-guidelines-for.html

Reply:
The first thing I saw that made me think of this was that `mkTelemetryCtx` wasn't in the same module as `data TelemetryCtx`.  I didn't know what I was looking at for a minute or so until I realized it was imported.